bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -340,6 +345,52 @@ impl Room {
         });
     }
 
+    pub fn create_poll(
+        &self,
+        question: String,
+        answers: Vec<String>,
+        max_selections: u64,
+        poll_kind: PollKind,
+        txn_id: Option<String>,
+    ) -> Result<(), ClientError> {
+        let timeline = match &*RUNTIME.block_on(self.timeline.read()) {
+            Some(t) => Arc::clone(t),
+            None => {
+                return Err(anyhow!("Timeline not set up, can't send the poll").into());
+            }
+        };
+
+        let options: Vec<UnstablePollAnswer> = answers
```

Comment:
I'd call this answers to to not have our product terminology (i.e. "options") pollute the matrix terminology ("answers").
This will shadow the answer param but AFAIU shadowing is idiomatic in Rust :)

Reply:
Addressed [here](https://github.com/matrix-org/matrix-rust-sdk/pull/2391/commits/482d690b8618839bd9ba3ea83fe74852904117c1) by changing the name of the variable